pandas/tests/test_multilevel.py

```diff
@@ -1490,6 +1490,16 @@ def test_frame_dict_constructor_empty_series(self):
         DataFrame({"foo": s1, "bar": s2, "baz": s3})
         DataFrame.from_dict({"foo": s1, "baz": s3, "bar": s2})
 
+    @pytest.mark.parametrize("d", [4, "d"])
+    def test_empty_frame_groupby_dtypes_consistency(self, d):
+        # GH 20888
+        group_keys = [...]
```

Comment:
```suggestion
        group_keys = ["a", "b", "c"]
```

I used ellipses as shorthand to indicate you use `group_keys = ["a", "b", "c"]`

Reply:
Sorry - my bad. I put back the correct `group_keys`.